### PR TITLE
[minor] couple of quick cleanups for kill reporoot

### PR DESCRIPTION
**Pull request by jayunit100**
quick fixes from last one,  cc @spxtr @ixdy follow on to  https://github.com/kubernetes/kubernetes/pull/25584

### CODE REVIEW

**Comment by ixdy** [comment]
LGTM, thanks!

**Comment by spxtr** [comment]
The bash change LGTM.

File: hack/update-bindata.sh [hunk 1]
```diff
@@ -16,14 +16,13 @@
 
 set -o errexit
 set -o pipefail
+set -o nounset
 
-if [[ -z ${KUBE_ROOT} ]]; then
+if [[ -z "${KUBE_ROOT:-}"]]; then
```
Comment (ixdy) [inline hack/update-bindata.sh]: oops. I think you're missing a space between `"` and `]]`

**Comment by jayunit100** [comment]
needs new lgtm i think.

**Pull request merged**